src/index.html

```diff
@@ -9,6 +9,31 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="logo">
+        <a href="#home">
+          <img src="images/logo.png" alt="logo">
+        </a>
+      </div>
+
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="list_item"><a href="#" class="list_a is-active">Apple</a></li>
```

Comment:
```suggestion
          <li class="list_item">
            <a href="#" class="list_a is-active">
              Apple
            </a>
          </li>
```
prettify your list markup 😉